api_testing/apis/test_plan_api.py

```diff
@@ -301,21 +300,22 @@ def create_testplan_from_test_unit_id(self, test_unit_id):
         testplan, payload = self.create_testplan(testUnits=[formated_testunit],
                                                  selectedArticles=[formatted_article],
                                                  materialType=[formatted_material])
-        if testplan['status'] == 1:
-            return (self.get_testplan_form_data(id=testplan['testPlanDetails']['id']))
+        if testplan['message'] == 'operation_success':
+            return self.get_testplan_form_data(id= payload['number'])
         else:
             self.info(testplan)
 
     def create_testplan_with_article_not_all(self):
         response, _ = GeneralUtilitiesAPI().list_all_material_types()
         formatted_material = random.choice(response['materialTypes'])
+
         formatted_article = ArticleAPI().get_formatted_article_with_formatted_material_type(formatted_material)
         testplan, payload = self.create_testplan(selectedArticles=[formatted_article],
                                                  materialType=[formatted_material])
-        if testplan['status'] == 1:
-            return (self.get_testplan_form_data(id=testplan['testPlanDetails']['id']))
+        if testplan['message'] == 'operation_success':
+            return (self.get_testplan_form_data(id=payload['number']))
         else:
-            self.info(testplan)
+            raise Exception(f'cant create the test plan with payload {payload}')
```

Comment:
raise Exception('cant create the test plan with payload {payload}')

Reply:
what do u mean?

Reply:
what is "f" letter before "cant..."?